### DISCUSSION TITLE
HG08010 not configurable

### DISCUSSION THREAD

**Discussion by RVis78** [Koenkk/zigbee2mqtt#13287] [Devices]
Dear Koenkk

Yesterday i bought the LIDL LED floodlight (HG08010).

I was able to pair the device with mqtt and also renamed it to Floodlight (friendly_name).
For some strange reason it is not configurable. In MQTT the following is listed:

![afbeelding](https://user-images.githubusercontent.com/62797454/180870297-ae574d7a-b50c-427a-b08e-13420e949208.png)

What can I further do to configure the LED Floodlight to control in in HA?

**Comment by Koenkk** [comment]
This device is supported in the latest dev branch and will be in the next release on 1 August.  (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by bri-tz** [reply]
i have this device but i have a strange problem with it.
it turns itself on several times a day.
if I look at the status of state, then it is off. but if I press the update button it turns on.
is there any way i can get z2m to poll status every 5 min?

**Comment by RVis78** [comment]
Hi Koen,

Thanks for your reply on my question. I will try the latest dev branch.



Met vriendelijke groet,

Remco Visser
